src/transactions/TransactionFrame.cpp

```diff
@@ -836,7 +836,9 @@ TransactionFrame::applyOperations(SignatureChecker& signatureChecker,
             << "Unknown exception while applying operations (txHash= "
             << xdr_to_string(getFullHash()) << ")";
     }
-
+#ifdef FUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION
```

Comment:
This doesn't belong here. You should check if the result was `txINTERNAL_ERROR` in `FuzzTransactionFrame::attemptApplication` and abort there.

Reply:
sure